publication-commons/src/test/resources/features/file_access_write.feature

```diff
@@ -0,0 +1,49 @@
+@ignore
+Feature: File metadata write and file delete permissions
+  As a system user
+  I want file metadata write (editing and deletion) to have clear permissions
+  So that only authorized users can perform those actions based on the file state
+
+  Background:
+  The publication metadata is published. X is an affiliation and this affiliation is given by your login context.
+
+  Scenario Outline: Verify file write permissions
+    Given a file in the "<FileState>" state
+    When a user with the role "<UserRole>"
+    And the user attempts to edit or delete the file
+    Then the action outcome is "<Outcome>"
```

Comment:
I think it would be "easier" to maintain and more readable with the same structure for all three cucumber test classes if we combine `When` and `And` lines:

```
Scenario Outline: Verify file upload permissions
    Given a file in the <FileState> state
    When a user with the role <UserRole> attempts to <FileOperation>
    Then the action outcome is <Outcome>
```